reframe/frontend/cli.py

```diff
@@ -1240,11 +1240,11 @@ def module_unuse(*paths):
             try:
                 rt.modules_system.load_module(**m, force=True)
             except errors.EnvironError as e:
-                printer.warning(
-                    f'could not load module {m["name"]!r} correctly; '
-                    f'skipping...'
+                printer.error(
+                    f'could not load module {m["name"]!r} correctly'
                 )
-                printer.debug(str(e))
+                printer.error(str(e))
```

Comment:
I'd print that in verbose mode. It's too much info for printing all this right away. You can also extend the error message with a `rerun with increased verbosity for more information`.